src/sparsezoo/model/model.py

```diff
@@ -739,11 +738,35 @@ def _sample_outputs_list_to_dict(
     def _get_directory_from_loose_api_files(
         files, directory_class, display_name, parent_directory, **kwargs
     ):
-        # fetch all the loose files that belong to the directory (use `file_type` key
+        # fetch all the loose files that belong to the directory (use `display_name` key
         # from the `request_json` for proper mapping)
-        files = [
-            file_dict for file_dict in files if file_dict["file_type"] == display_name
+        files_by_display_name = [
```

Comment:
call model.deployment_directory_path:
 -> 1. No local deployment or deployment.tar.gz --> download only deployment.tar.gz and untar 
 -> 2. There is local deployment extracted > simply create model.directory: SelectDirectory(files=...)
 -> 3. There is a local deployment archived-> untar and create model.directory: SelectedDirectory(files=...)